packages/terra-divider/src/terra-dev-site/test/divider/DefaultDivider.test.jsx

```diff
@@ -2,4 +2,10 @@ import React from 'react';
 
 import Divider from '../../../Divider';
 
-export default () => <Divider id="divider" />;
+export default () => (
+  <div>
+    <div style={{ height: '2rem', background: '#40E0D0' }} />
```

Comment:
could this cause color contrast issues with theme development? if so, we should create theme variables for the example background colors

Reply:
No text is rendered on these. These divs exist to highlight the box-model of the hr element and the hr elements related margin.